.pre-commit-config.yaml

```diff
@@ -33,3 +33,13 @@ repos:
     hooks:
       - id: eslint
         args: ["monkey/monkey_island/cc/ui/src/", "--fix", "--max-warnings=0"]
+  - repo: local
```

Comment:
I made a small performance improvement in `pyproject.toml` that shaved 6 seconds off the pytest runtime; it now runs in 2.5 seconds on my rig. IMHO, an additional 2.5sec is still a bit too cumbersome for each commit. Could we run this on push with `stages: push` instead of on each commit?